ui/apps/platform/src/Containers/ComplianceEnhanced/ScanConfigs/Wizard/ScanConfigWizardForm.tsx

```diff
@@ -4,19 +4,27 @@ import { Wizard, WizardStep } from '@patternfly/react-core';
 
 import { complianceEnhancedScanConfigsBasePath } from 'routePaths';
 
+import { FormikProvider } from 'formik';
+import useFetchClustersForPermissions from 'hooks/useFetchClustersForPermissions';
 import ScanConfigOptions from './ScanConfigOptions';
 import ClusterSelection from './ClusterSelection';
+import ProfileSelection from './ProfileSelection';
 import ScanConfigWizardFooter from './ScanConfigWizardFooter';
 import useFormikScanConfig from './useFormikScanConfig';
 
 const PARAMETERS = 'Set Parameters';
 const PARAMETERS_ID = 'parameters';
 const SELECT_CLUSTERS = 'Select clusters';
 const SELECT_CLUSTERS_ID = 'clusters';
+const SELECT_PROFILES = 'Select profiles';
+const SELECT_PROFILES_ID = 'profiles';
 
 function ScanConfigPage(): ReactElement {
     const history = useHistory();
     const formik = useFormikScanConfig();
+    const { clusters, isLoading: isFetchingClusters } = useFetchClustersForPermissions([
+        'Compliance',
+    ]);
```

Comment:
I think we need to replace this call with a call to the API endpoint Core Workflows is doing which returns a list of clusters which are set up for Compliance 2.0 (that is, OpenShift + Compliance Operator installed).

@dashrews78 Can you confirm which endpoint this is? (I am missing it in the API doc.)

Reply:
`/v2/compliance/integrations`

In the data model and API doc it is under the heading `CO Metadata`